README.md

```diff
@@ -89,6 +89,36 @@ output "packages" {
 }
 ```
 
+Download the raw file for a package, the package version should equal to the package version visible in UI:
```

Comment:
Rather than an example in the readme, if you add a real docs page (check out the other resources) then it'll show those docs on the terraform provider site when released.

An example here will only show on GitHub and be hard for most users to find.